listenbrainz/db/stats.py

```diff
@@ -0,0 +1,55 @@
+"""This module contains functions to insert and retrieve statistics
+   calculated from Google BigQuery into the database.
+"""
+
+import sqlalchemy
+import ujson
+from listenbrainz import db
+
+
+# XXX(param): think about the names of these stats variables
```

Comment:
https://github.com/metabrainz/listenbrainz-server/pull/244/files/b9e6120692cc688b3d8e1867918911350893a66e#diff-d4071fbd1594577acc0c16faf1b47578R14

Which is more appropriate `artists` or `artist_stats`?